spec.bs

```diff
@@ -358,6 +362,8 @@ const request = new PaymentRequest([{
     // List of credential IDs obtained from the bank.
     credentialIds,
 
+    rpId: "https://fancybank.com/",
```

Comment:
If `rpId` is a domain, should this be just `fancybank.com`? I'm not too familiar with WebAuthn, so might be worth testing this out and seeing!

Reply:
Ah yep, it should just be fancybank.com (confirmed with webauthn enrollment, at least). Fixed.